Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -58,21 +58,21 @@ def test_font(self):
         dfont = idleConf.GetFont(root, 'main', 'EditorWindow')
         dsize = str(dfont[1])
         dbold = dfont[2] == 'bold'
-        configure.fontName.set('Test Font')
+        configure.font_name.set('Test Font')
```

Comment:
`dfont`, `dsize`, `dbold`, may changed explicited to `default_font`, `default_size`, `default_bold` since it is testing about default settings.